libs/core/langchain_core/messages/ai.py

```diff
@@ -253,14 +253,23 @@ def init_tool_calls(cls, values: dict) -> dict:
                 else:
                     raise ValueError("Malformed args.")
             except Exception:
-                invalid_tool_calls.append(
-                    InvalidToolCall(
-                        name=chunk["name"],
-                        args=chunk["args"],
-                        id=chunk["id"],
-                        error=None,
+                if chunk["args"] == "":
+                    tool_calls.append(
+                        ToolCall(
+                            name=chunk["name"] or "",
+                            args={},
+                            id=chunk["id"],
+                        )
+                    )
```

Comment:
should we add this case before the exception gets thrown? maybe in the `args_ = parse_partial... if chunk["args"] != "" else {}` or something